README.md

```diff
@@ -61,11 +61,11 @@ Now you are good to go!
 ### repo approve
 
 ```sh
-$ repo approve [regex]
+$ repo approve [--title title]
 ```
 
-Iterates over all open pull requests matching `regex` (all PRs if
-no regex is given) in all configured repositories.
+Iterates over all open pull requests matching `title` (all PRs if
+no regex for title is given) in all configured repositories.
```

Comment:
to my comment, perhaps this is enough? I guess `--title` is just a replacement to the positional?

Reply:
-  does it still accept a regex?
- Yes, `--title title` still works as a regex, only the name and option changed.

Reply:
Polished README, thanks for the suggestions!
And yes `--title` is a positional flag for regex. when I first used this command, I feel confused about how to set regex and tried to use `repo tag --branch branch 'tagName'` but regex is required even if the title name is provided. so the command is like this: `repo tag --branch branch . 'tagName'`. I thought it will be very helpful if we explicit set the regex as a flag, then user can `repo tag --branch brahch --title 'tagName'` it might be clearer. 
I was targeting at `repo tag itself` and then to make all command set aligned, all commands are changed to `--title`  option. WDYT?